src/components/InventoryDetail/ApplicationDetails.js

```diff
@@ -105,9 +105,9 @@ const ApplicationDetails = ({
           ))}
         </Tabs>
       </section>
-      <section>
-        {activeTabs?.length > 0 &&
-          activeTabs?.map((item) => {
+      {activeTabs?.length > 0 ? (
```

Comment:
The condition could already start on line 80, and wrap really all of the component. Then the odd "<>" when it loads form the tabs should be gone too.